test/files/change-sets/prod-faktory.md

```diff
@@ -0,0 +1,27 @@
+:warning: This PR generates **23** changes for `some-stack`.
```

Comment:
I might find it a tad confusing to have the input `.json` be in the same directory as the outputs

Reply:
Yeah, I think there's precedent for both ways. In our frontend apps `a/b/c.x` becomes `/a/b/__snapshots__/c.x.snap`. `cfn-flip` puts them [side-by-side like this](https://github.com/awslabs/aws-cfn-template-flip/tree/master/examples) (where I got the idea).

I think I lean this way because conceptually, every `foo.*` are the same thing under different representations. Putting them side-by-side makes sense (to me). This is generally useful beyond the golden testing, which is the only thing imparting some kind of input-vs-outputs idea on them.